pkg/cri/sbserver/podsandbox/sandbox_stop.go

```diff
@@ -44,7 +45,7 @@ func (c *Controller) Stop(ctx context.Context, sandboxID string) (*api.Controlle
 	// TODO: The Controller maintains its own Status instead of CRI's sandboxStore.
 	// Only stop sandbox container when it's running or unknown.
 	state := sandbox.Status.Get().State
-	if state == sandboxstore.StateReady || state == sandboxstore.StateUnknown {
+	if (state == sandboxstore.StateReady || state == sandboxstore.StateUnknown) && sandbox.Container != nil {
```

Comment:
Do we need `sandbox.Container != nil` in cri/server/sandbox_stop?

Reply:
Nope, `cri/server/sandbox_stop.go` doesn't dereference `sandbox.Container`.